quiz-bundle-spec.md

```diff
@@ -0,0 +1,71 @@
+# Qwiklabs Quiz Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it easier to see the general file structure.
+
+```yml
+entity_type: Quiz
+schema_version: 1
+
+# The authoring title of the quiz - not visible to learners.
+title: Super Hard Quiz
+
+# The threshold grade that a student needs to achieve in order to count as "passing" the quiz.
+passing_percentage: 67
+
+# The "items" (i.e. questions) contained in this quiz.
+items: ...
+```
+
+Note that all of the localized content (stem, option titles, rationales, etc) are HTML content that may be displayed in various contexts. All of these chunks will be sanitized according to the restricted set in the [HTML spec](./html-spec.md).
+
```

Comment:
Add a full specification of the top level attributes, a la [course bundle spec](https://github.com/CloudVLab/qwiklabs-content-bundle-spec/blob/master/course-template-bundle-spec.md#attribute-specification)